src/checkers/gnomechecker.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from urllib.parse import urljoin
+import typing as t
+
+import requests
+
+from ..lib.externaldata import Checker, ExternalData, ExternalFile
+
+log = logging.getLogger(__name__)
+
+GNOME_MIRROR = "https://download.gnome.org/"
+
+
+def _parse_checksums(text: str) -> t.Dict[str, str]:
+    result = {}
+    for line in text.splitlines():
+        digest, filename = line.strip().split(maxsplit=1)
+        result[filename] = digest
+    return result
+
+
+def _is_stable(version: str) -> bool:
+    major, minor = version.split(".")[:2]
+    if int(major) >= 40:
+        return minor not in ["alpha", "beta", "rc"]
+    return (int(minor) % 2) == 0
```

Comment:
I'm not sure about the algorithm to check if given version is stable. Is it ok to assume that all versions prior to 40 had minor version number being odd for pre-releases?